docs/management/watcher-ui/index.asciidoc

```diff
@@ -1,8 +1,56 @@
 [role="xpack"]
 [[watcher-ui]]
-== Watcher UI
+== Watcher
+
+Watcher is an {es} feature that you can use to create actions based on 
+conditions, which are periodically evaluated using queries on your data. 
+Watches are helpful for analyzing mission-critical and business-critical 
+streaming data. For example, you might watch application logs for performance 
+outages or audit access logs for security threats.
+
+With the Watcher UI, you can:
+
+* Create a simple threshold watch
+* View your watch history and action status
+* Acknowledge a watch to prevent too many executions
+* Edit, deactivate, and delete a watch
+* Create more advanced watches using API syntax
+
+[role="screenshot"]
+image:management/watcher-ui/images/watches.png["Watcher list"]
+
+{stack-ov}/xpack-alerting.html[Alerting on cluster and index events]
+is a good source for detailed 
+information on how watches work. If you are using the UI to create a 
+threshold watch, take a look at the different watcher actions. If you are 
+creating an advanced watch, you should be familiar with the parts of a 
+watch&#8212;input, schedule, condition, and actions.
+
+[float]
+[[watcher-security]]
+=== Required roles for Watcher
+
+To use Watcher, you must have the built-in `kibana_user` role.  
+You must also have one of the following Watcher roles:
+
+* `watcher_admin` can perform all Watcher actions, such as creating a new watch 
+or editing an existing one.
+
+* `watcher_user` can view all existing watches, but cannot create or edit watches.
+
+You can manage roles in *Management > Security > Roles*, or use the 
+<<role-management-api, Kibana Role Management API>>. Watches are shared between 
+all users with the same role.  
```

Comment:
When we're describing security requirements, we often start by saying which privileges they require (e.g. manage_watcher or monitor_watcher cluster privileges per https://www.elastic.co/guide/en/elastic-stack-overview/master/security-privileges.html . Then we mention the built-in roles in case they don't want to add these privileges to their existing roles.